src/qtt/instrument_drivers/virtualAwg/virtual_awg.py

```diff
@@ -182,15 +182,19 @@ def disable_outputs(self, gate_names):
             A start command is required to enable the outputs.
 
         Arguments:
-            gate_names (list) The names of the gates which needs to be disabled.
+            gate_names (list or None) The names of the gates which needs to be disabled. If None, then disable all gates
         """
-        if VirtualAwg.__digitizer_name in self._settings.awg_map:
-            gate_names.extend([VirtualAwg.__digitizer_name])
-        if VirtualAwg.__awg_slave_name in self._settings.awg_map:
-            gate_names.extend([VirtualAwg.__awg_slave_name])
-        for name in gate_names:
-            (awg_number, channel_number, *_) = self._settings.awg_map[name]
-            self.awgs[awg_number].disable_outputs([channel_number])
+        if gate_names is None:
+            for awg in self.awgs:
+                awg.disable_outputs(None)
```

Comment:
None not needed here anymore when the other comment is resolved.
```suggestion
                awg.disable_outputs()
```